heartbeat/beater/heartbeat.go

```diff
@@ -182,10 +182,8 @@ func (bt *Heartbeat) RunStaticMonitors(b *beat.Beat) (stop func(), err error) {
 
 // RunCentralMgmtMonitors loads any central management configured configs.
 func (bt *Heartbeat) RunCentralMgmtMonitors(b *beat.Beat) {
-	mons := cfgfile.NewRunnerList(management.DebugK, bt.dynamicFactory, b.Publisher)
-	reload.Register.MustRegisterList(b.Info.Beat+".monitors", mons)
 	inputs := cfgfile.NewRunnerList(management.DebugK, bt.dynamicFactory, b.Publisher)
-	reload.Register.MustRegisterList("inputs", inputs)
+	reload.RegisterV2.MustRegisterList("inputs", inputs)
```

Comment:
Should this be `"input"` (as referenced in https://github.com/elastic/beats/pull/32673/files#diff-a8a81a292d2ae57f29639278b79456bc3a0106cab524e21cb40150a974391591R30) or is `"inputs"` still valid (also used by apm-server currently)?

Reply:
So, the name itself is irrelevant, and it's mostly a holdover from me reusing the V1 APIs. As long as all the V2 beats/inputs are using the same name, it'll work.

Reply:
Can we just eliminate the name argument then to avoid confusion?

Reply:
Yah, agreed, probably best to just get rid of that.